osh.spec

```diff
@@ -241,13 +244,16 @@ fi
 %{_sbindir}/osh-retention
 %{_sbindir}/osh-stats
 %{_sysconfdir}/osh/hub
-%exclude %{python3_sitelib}/osh/hub/scripts/osh-xmlrpc-client.py
-%exclude %{python3_sitelib}/osh/hub/scripts/umb-emit.py
+%exclude %{python3_sitelib}/osh/hub/scripts/osh-xmlrpc-client.py*
+%exclude %{python3_sitelib}/osh/hub/scripts/umb-emit.py*
 %{python3_sitelib}/osh/hub
 %{_unitdir}/osh-retention.*
 %{_unitdir}/osh-stats.*
 %exclude %{python3_sitelib}/osh/hub/settings_local.py*
+%exclude %{python3_sitelib}/osh/hub/settings_local.ci.py*
+%exclude %{_sysconfdir}/osh/hub/settings_local.py*
 %exclude %{python3_sitelib}/osh/hub/__pycache__/settings_local.*
+%exclude %{python3_sitelib}/osh/hub/__pycache__/settings.*
```

Comment:
I think this line was added by mistake.  `settings.py` is still owned by `osh-hub` and so should be the corresponding byte-code.